airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -437,6 +438,11 @@ def test_filter_assets_by_uri_pattern_works(self, test_client, params, expected_
     def test_filter_assets_by_dag_ids_works(self, test_client, dag_ids, expected_num, session):
         session.query(DagModel).delete()
         session.commit()
+        bundle_name = "test_bundle"
+        orm_dag_bundle = DagBundleModel(name=bundle_name)
```

Comment:
You can probably simplify a lot of these tests using the [testing_dag_bundle fixture](https://github.com/apache/airflow/blob/5de7cda70478ff43c3b7fb995092900750fb7996/devel-common/src/tests_common/pytest_plugin.py#L2436).